docs/package_index_json-specification.md

```diff
@@ -194,9 +196,9 @@ Each PLATFORM describes a core for a specific architecture. The fields needed ar
 * `architecture`: is the architecture of the plaftorm (avr, sam, etc...). It must match the architecture of the core as explained in the [Arduino platform specification](platform-specification.md#hardware-folders-structure)
 * `version`: the version of the platform.
 * `category`: this field is reserved, a 3rd party core must set it to `Contributed`
-* `help`/`online`: is a URL that is displayed on the Boards Manager as an "Online Help" link
+* `help`/`online`: is a URL that is displayed on the Arduino IDE's Boards Manager as an "Online Help" link
 * `url`, `archiveFileName`, `size` and `checksum`: metadata of the core archive file. The meaning is the same as for the TOOLS
-* `boards`: the list of boards supported (note: just the names to display on the GUI! the real boards definitions are inside `boards.txt` inside the core archive file)
+* `boards`: the list of boards supported (note: just the names to display on the Arduino IDE/Pro IDE's Boards Manager GUI! the real boards definitions are inside `boards.txt` inside the core archive file)
 * `toolsDependencies`: the tools needed by this core. Each tool is referenced by the triple (`packager`, `name`, `version`) as previously said. Note that you can reference tools available in other packages as well.
 
 The `version` field is validated by both Arduino IDE and [JSemVer](https://github.com/zafarkhaja/jsemver). Here are the rules Arduino IDE follows for parsing versions ([source](https://github.com/arduino/Arduino/blob/master/arduino-core/src/cc/arduino/contributions/VersionHelper.java)):
```

Comment:
Go as you said, changing to _generic Arduino dev software_ or the like :+1:

Reply:
This statement only applies to Arduino IDE and Arduino Pro IDE. This information is never displayed by Arduino CLI and Arduino Web Editor.

Reply:
Then go ahead with _Arduino IDE and Arduino Pro IDE_ :+1:

Reply:
I have now made the suggested change: https://github.com/arduino/arduino-cli/compare/f6d65a6cd3b34ac109ac3627237a09d9a2b4528a..98334b87366ecd5360bf904e8e13dbb4c0ab675c